YololCompetition/Services/Schedule/InMemoryScheduler.cs

```diff
@@ -56,24 +58,50 @@ public async Task Start()
                 // If there is no challenge running try to start a new one
                 if (current == null)
                 {
-                    Console.WriteLine("There is no current challenge - attempting to start a new one");
+                    Console.WriteLine("There is no current challenge - checking if challenge finished within the last 24 hours");
 
-                    // Start the next challenge, if there isn't one wait a while
-                    var next = await _challenges.StartNext();
-                    if (next == null)
+                    var currentTime = DateTime.UtcNow;
+                    var startTime = (currentTime.Date).AddMinutes(_config.ChallengeStartTime);  //Gets the time today that the challenge would be starting if it were starting today
```

Comment:
As a matter of style I prefer `thing + TimeSpan.FromMinutes(N)` over `thing.AddMinutes(N)`. This is because it's not clear if `AddMinutes` mutates `thing` or not. Not critical though, just a style choice.

Reply:
This is why I included that () around currentTime.Date, to imply that it does not mutate the original thing. I understand your point though, and agree in principle. I'll change this and the other occurrences of similar.